clib/clib.go

```diff
@@ -1158,10 +1225,6 @@ func XMLChildNodes(n PtrSource) ([]uintptr, error) {
 	return ret, nil
 }
 
-type stringer interface {
```

Comment:
Yes, I get it, it's unused. But please don't mix things in a PR for something else.

Reply:
Reverted